Automattic/wp-calypso
# Create Jetpack product cards for each product type

### Changes proposed in this Pull Request

This PR adds the missing product cards (for the _Plan_ and _Individual Product_ products), and document the props.

### Implementation notes

- All cards are now grouped in the `components/jetpack/card/` folder.
- I moved `components/jetpack-product-card` to `components/jetpack/card/jetpack-product-card` without making any changes to the files. Feel free to skip this folder in your review.
- The nomenclature for "individual products" can be a bit confusing, since we now use "product" to encompass "plans", "bundles", and "individual products". Feel free to suggest something else.

### Testing instructions

1. Download the PR
2. Check that the header border, the price stroke, and the badge have the proper colour (not the icon), for each of these components in the dev docs (see screenshots below):
    - `devdocs/design/jetpack-plan-card`
    - `devdocs/design/jetpack-bundle-card`
    - `devdocs/design/jetpack-individual-product-card`
3. Check the README at `components/jetpack/card/jetpack-product-card/README.md`

### Screenshots

#### Plan Card
<img width="579" alt="Screen Shot 2020-08-04 at 4 25 57 PM" src="https://user-images.githubusercontent.com/1620183/89341483-62788180-d66f-11ea-8d1d-3c639861160a.png">
<img width="572" alt="Screen Shot 2020-08-04 at 4 26 02 PM" src="https://user-images.githubusercontent.com/1620183/89341486-63a9ae80-d66f-11ea-9508-14733e665084.png">


#### Bundle Card
<img width="571" alt="Screen Shot 2020-08-04 at 4 26 52 PM" src="https://user-images.githubusercontent.com/1620183/89341507-6ad0bc80-d66f-11ea-82fa-e72f32f99a77.png">
<img width="575" alt="Screen Shot 2020-08-04 at 4 26 56 PM" src="https://user-images.githubusercontent.com/1620183/89341510-6b695300-d66f-11ea-83a9-a7bbe931ccb6.png">


#### Individual Product Card
<img width="579" alt="Screen Shot 2020-08-04 at 4 26 22 PM" src="https://user-images.githubusercontent.com/1620183/89341541-77edab80-d66f-11ea-8a25-106ce7a160dd.png">
<img width="576" alt="Screen Shot 2020-08-04 at 4 26 26 PM" src="https://user-images.githubusercontent.com/1620183/89341542-77edab80-d66f-11ea-8c4c-b00bf5f0b205.png">
